stlab/forest_algorithms.hpp

```diff
@@ -0,0 +1,122 @@
+/**************************************************************************************************/
+
+#ifndef STLAB_FOREST_ALGORITHMS_HPP
+#define STLAB_FOREST_ALGORITHMS_HPP
+
+/**************************************************************************************************/
+
+// stdc++
+#include <optional>
```

Comment:
We have still in stlab stlab::optional, because - at least in the past, and it may have improved in the meantime, - Apple's implementation of the standard did not include optional, even the compiler supported C++17.
stlab::optional is a typedef on std::optional or boost::optional, depending on its availability.

Reply:
Fixed